src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx

```diff
@@ -50,6 +66,7 @@ const OfflineKeyShareGeneration = () => {
     });
 
     const cliCommand = `--operator-keys=${operatorsKeys.join(',')} --operator-ids=${operatorsIds.join(',') } --owner-address=${accountAddress} --owner-nonce=${ownerNonce}`;
+    const dkgCliCommand = `docker run -it "ssv-dkg:latest" /app init --owner ${walletStore.accountAddress} --nonce ${ownerNonce} --withdrawAddress ${withdrawalAddress} --network ${apiNetwork} --operatorIDs ${operatorsIds.join(',')} --depositResultsPath /data --ssvPayloadResultsPath /data --operatorsInfo ${JSON.stringify(Object.values(operatorStore.selectedOperators))} --encryptedKeyOutputPath /data --generateKeypair `;
```

Comment:
It is interesting how /data folder is handled. 
Where is it, should it be created before or will be created automatically. Looks like it's about folder in docker volume associated with container and mounted to host